SmplEditor/MainWindow.xaml.cs

```diff
@@ -338,39 +397,48 @@ private void OnSortOptionChanged(object sender, SelectionChangedEventArgs e)
             int selectedOption = SortOptionComboBox.SelectedIndex;
             switch (selectedOption){
                 case 0: // by Order, directory
+                // current structure of <Song> has a problem in this
+                // Turns out, each song has its own "order" field per playlist
+                // Currently, the "order" is considered as a <Song> object's property.
+                // It's actually a property of each playlist with mapping to the songs.
                     {
+                        // A song library does not have duplicates nor its own ordering
+                        // (since ordering is an attribute of a playlist)
+                        // There needs to be an alternative way to sort.
+                        // The best I can think of now is by artist-title.
+                        // Another Idea is Dir-title. Or jsut full path.
```

Comment:
This still needs to be fixed. `Sort by order` in AllSongs option listing.
Under Song.cs, the smpl<->smpl case is handled, but not perfectly, since (look SmplSong.cs) the &lt;order&gt; attribute of a smpl song is not a good representation of a smpl song instance. (It's more like a playlist attribute given to a song)
smpl <-> iTunes is not implemented.
iTunes <-> iTunes is not implemented.

Reply:
issue #7 posted